pkg/util/kubeconfig_test.go

```diff
@@ -394,6 +394,102 @@ func TestDeleteKubeConfigContext(t *testing.T) {
 	}
 }
 
+func TestSetCurrentContext(t *testing.T) {
+	kubeConfigFile := "./testdata/kubeconfig/config"
+	contextName := "minikube"
+
+	cfg, err := ReadConfigOrNew(kubeConfigFile)
+	if err != nil {
+		t.Fatalf("Error not expected but got %v", err)
+	}
+
+	if cfg.CurrentContext != "" {
+		t.Errorf("Expected empty context but got %v", cfg.CurrentContext)
+	}
+
+	err = SetCurrentContext(kubeConfigFile, contextName)
```

Comment:
Rather than mutating a path within the git tree, can this use copy the file elsewhere, perhaps using `ioutil.TempFile` or a TempDir?

Reply:
done